.github/workflows/build.yml

```diff
@@ -35,6 +35,18 @@ jobs:
           file: ./out/tests/coverage-unit.txt
           flags: unit,os_linux
           fail_ci_if_error: true
+  test-linux-arm64:
+    runs-on: linux-arm64
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+      - name: Test
+        run: |
+          make format || true
```

Comment:
`make format` fails on the arm machine... the build directives might need to be updated. In the interest of time, I made this a part of #697.